packages/app-project/src/screens/ClassifyPage/components/CollectionsModal/CollectionsModal.js

```diff
@@ -0,0 +1,42 @@
+import counterpart from 'counterpart'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { Grid, Grommet } from 'grommet'
+import zooTheme from '@zooniverse/grommet-theme'
+import { Modal } from '@zooniverse/react-components'
+import styled from 'styled-components'
+
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+function CollectionsModal ({
+  active,
+  children,
+  closeFn
+}) {
+  return (
+    <Modal
+      active={active}
+      closeFn={closeFn}
+      title={counterpart('CollectionsModal.title')}
+    >
+      <Grommet>
```

Comment:
The `Grommet` component is meant to be top level for merging themes and injecting into context, so it shouldn't be needed here. Did you add this to try to get styles working? Perhaps #530 should merge first.

Reply:
Yeah, this is leftover from getting basic styling to work.